st2reactor/st2reactor/rules/filter.py

```diff
@@ -61,6 +61,8 @@ def filter(self):
                  extra=self._base_logger_context)
 
         if not self.rule.enabled:
+            if self.extra_info:
+                LOG.info('Validation failed for rule %s as it is disabled.', self.rule.ref)
```

Comment:
`extra_info` is for extra logs needed for rule tester to make sense. 

In this case a rulebbeing skipped because it is disabled is useful for rule tester but not for standard ruleengine logs.